src/PrestaShopBundle/Resources/views/Admin/Sell/Order/Order/preview.html.twig

```diff
@@ -139,7 +139,13 @@
           <tbody>
           {% for productDetail in orderPreview.productDetails %}
             <tr class="{% if loop.index > productsPreviewLimit %}js-product-preview-more d-none{% endif %}">
-              <td class="p-1"><a class="px-0 external-link" href="{{ path('admin_product_form', {'id': productDetail.id}) }}" target="_blank">{{ productDetail.name }}</a></td>
+              <td class="p-1">
+                {% if productDetail.id %}
+                  <a class="px-0 external-link" href="{{ path('admin_product_form', {'id': productDetail.id}) }}" target="_blank">{{ productDetail.name }}</a>
+                {% else %}
+                  {{ productDetail.name }} ({{ 'Product deleted'|trans({}, 'Admin.Global') }})
```

Comment:
```suggestion
                  {{ productDetail.name }} ({{ 'Product deleted'|trans({}, 'Admin.Catalog.Feature') }})
```

Reply:
Hello @lartist / @jolelievre , where does this string appear? I'm not sure I understand what 'product deleted' means (whether it's a notification or just a text somewhere)

Reply:
Hi @lartist 

this one is not a notification it's displayed in the Order details when a product from the order has been removed:
![Capture d’écran 2023-02-03 à 20 09 08](https://user-images.githubusercontent.com/13801017/216686997-f645626d-e690-4580-aeb9-ff0746938839.png)

Reply:
> Hi @lartist
> 
> this one is not a notification it's displayed in the Order details when a product from the order has been removed: ![Capture d’écran 2023-02-03 à 20 09 08](https://user-images.githubusercontent.com/13801017/216686997-f645626d-e690-4580-aeb9-ff0746938839.png)

Sorry I'm lost, Which catalog do I have to change finally? :)